emma-language/src/test/scala/org/emmalanguage/compiler/lang/funsupport/normalizeTest.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2014 TU Berlin (emma@dima.tu-berlin.de)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.emmalanguage
+package compiler.lang.funsupport
+
+import compiler.MacroCompiler
+
+import scala.language.experimental.macros
+import scala.reflect.macros.blackbox
+
+object normalizeTest {
+  final def apply[T](e: T): String = macro normalizeTestMacro.parallelizeImpl[T]
+}
+
+class normalizeTestMacro(val c: blackbox.Context) extends MacroCompiler {
+
+  import FunSupport.CG
+
+  def parallelizeImpl[T](e: c.Expr[T]): c.Expr[String] = {
+    c.Expr[String](parallelize(e))
+  }
+
+  lazy val parallelize: c.Expr[Any] => u.Tree =
+    pipeline(typeCheck = false)(inine).compose(_.tree)
+
+  lazy val inine: u.Tree => u.Tree = tree => {
```

Comment:
Removed the whole file. The `inine` method is [now called `expand` and is part of the `LibSupport` API and is called `expand`](https://github.com/emmalanguage/emma/pull/253/commits/6a5390facaac77164f76939568254d99d5df76dc#diff-cd641dcce68f334f9299d35da1684e0cR123).